lib/modules/versioning/debian/index.ts

```diff
@@ -0,0 +1,134 @@
+import type { RangeStrategy } from '../../../types';
+import { DistroInfo, DistroInfoRecordWithVersion } from '../distro';
+import { GenericVersioningApi } from '../generic';
+import type { GenericVersion } from '../generic';
+import type { NewValueConfig, VersioningApi } from '../types';
+
+export const id = 'debian';
+export const displayName = 'Debian';
+export const urls = [
+  'https://debian.pages.debian.net/distro-info-data/debian.csv',
+];
+export const supportedRangeStrategies: RangeStrategy[] = ['pin'];
```

Comment:
```suggestion
export const supportsRanges = true;
export const supportedRangeStrategies: RangeStrategy[] = ['pin'];
```

That's why the metadata test fails 🙃

Reply:
☝️ ❓

Reply:
![image](https://user-images.githubusercontent.com/97394622/164776921-a7515759-29c6-42c5-bc09-574952f2a3e5.png)

^^ this supposed to be outdated 😶